certbot/tests/compat/filesystem_test.py

```diff
@@ -280,14 +280,14 @@ def test_makedirs_switch_os_mkdir(self):
         self.assertEqual(original_mkdir, std_os.mkdir)
 
 
-class OwnershipTest(test_util.TempDirTestCase):
+class CopyOwnershipAndModeTest(test_util.TempDirTestCase):
     """Tests about copy_ownership_and_apply_mode and has_same_ownership"""
```

Comment:
nit: We should update this comment since we're testing more functions here.

Reply:
Done.